cmd/orb-server/startcmd/start.go

```diff
@@ -625,16 +625,10 @@ func (m mockTxnProvider) RegisterForDID() <-chan []string {
 }
 
 const mockProof = `{
- "@context": [
-   "https://w3id.org/security/v1",
-   "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json"
- ],
- "mockProof": {
-   "type": "JsonWebSignature2020",
-   "proofPurpose": "assertionMethod",
-   "created": "2021-01-27T09:30:15Z",
-   "verificationMethod": "did:example:abcd#key",
-   "domain": "https://witness1.example.com/ledgers/maple2021",
-   "jws": "eyJ..."
- }
+   "algorithm":{
```

Comment:
The witness need to return an LD proof in activitypub. Where is this mock being used?

Reply:
@troyronda we need to discuss this part. 
`Where is this mock being used?` - currently, we are not using it.